app/k8s-api-client.js

```diff
@@ -22,16 +22,16 @@ exports.get_namespace = get_namespace;
 
 class KubernetesAPIClient extends APIClient {
   endpoint_url(endpoint, namespace = undefined) {
+    let ns = namespace || this.namespace || 'default';
+
     if (!endpoint) {
       throw new Error('Missing endpoint');
     }
 
-    let ns = namespace || this.namespace || 'default';
-
     const api = {
       'deployments': 'apis/apps/v1beta2',
       'pods': 'api/v1',
-    }[endpoint];
+    }[endpoint.split('/')[0]];
```

Comment:
I probably didn't read the relevant code yet but...why the split here? I've seen both `Deployment` and `Pod` `endpoint()` returns a string without `/`, what am I missing here?

Reply:
This is a bugfix - the endpoint passed in can be something like `deployment/andyhd-rstudio-rstudio`, but that doesn't exist in the `api` lookup object here.